stories/theme/ag/large/Modals/MoreOptionsModal.css

```diff
@@ -0,0 +1,199 @@
+  overflow-y: auto;
+}
+
+.MoreOptionsModal_InnerContainer {
+  background: #FFF;
+  display: flex;
+  flex-direction: column;
+  min-height: 300px;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+}
+
+.MoreOptionsModal .ModalBody > div {
+  overflow-y: scroll;
+  display: flex;
+  flex-direction: column;
+  max-height: MoreOptionsModalHeight;
+  min-height: 300px;
+  overflow-x: hidden;
+}
+
+.MoreOptionsModal .Modal-open {
+  display: block;
+}
+
+.MoreOptionsModal .Modal-close {
+  display: none;
+}
+
+.MoreOptionsModal_CloseButton {
+  float: right;
+  cursor: pointer;
+  font-family: NunitoSans;
+  font-size: 15px;
+  font-weight: bold;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  text-align: right;
+  color: #4a90e2;
+  margin-top: 28px;
+  margin-right: 40px;
+}
+
+.MoreOptionsModal_Header {
+  flex-shrink: 0;
+  flex-grow: 0;
+  margin-bottom: 10px;
+  display: flex;
+  justify-content: space-between;
+  box-shadow: 0 1px 3px 0 rgba(170, 170, 170, 0.5);
+}
+
+.MoreOptionsModal_Content {
+  flex-grow: 1;
+  overflow-y: scroll;
+  /* Prevent scrollbar from affecting the width */
+  /* Prevent horizontal scrollbar when price slider floats right */
+  overflow-x: hidden;
+  padding: 0 40px;
+}
+
+.MoreOptionsModal_Content::-webkit-scrollbar {
+  width: 7px;
+  background: transparent;
+}
+
+.MoreOptionsModal_Content:hover::-webkit-scrollbar-thumb {
+  border-radius: 100px;
+  background-color: #d8d8d8;
+  -webkit-box-shadow: 0 0 1px rgba(255, 255, 255, 0.5);
+}
+
+.MoreOptionsModal_PropertyCount {
+  font-family: NunitoSans;
+  font-size: 18px;
+  font-weight: bold;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4a4a4a;
+  margin-top: 26px;
+  margin-left: 40px;
+}
+
+.MoreOptionsModal_Footer {
+  box-shadow: 0 -1px 3px 0 rgba(170, 170, 170, 0.5);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-shrink: 0;
+  flex-grow: 0;
+  height: 110px;
+  padding: 0 40px;
+}
+
+.MoreOptionsModal_ResetAll {
+  color: #4a90e2;
+  font-family: NunitoSans;
+  font-size: 15px;
+  font-weight: bold;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  cursor: pointer;
+  background: none;
+  border: none;
+}
+
+.MoreOptionsModal_Cancel {
+  width: 200px;
+  height: 36px;
+  background: none;
+  padding: 0;
+  border-radius: 100px;
+  border: solid 1px #4a90e2;
+  font-family: NunitoSans;
+  font-size: 15px;
+  font-weight: bold;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  text-align: center;
+  color: #4a90e2;
+  cursor: pointer;
+  margin-right: 32px;
+}
+
+.MoreOptionsModal_ShowProps {
+  width: 200px;
+  height: 36px;
+  border-radius: 100px;
+  background-color: #4a90e2;
+  font-family: NunitoSans;
+  font-size: 15px;
+  font-weight: bold;
+  font-style: normal;
```

Comment:
Please remove all normal properties where it’s already the default.

Reply:
fixed